stores/sql/types.go

```diff
@@ -207,8 +207,13 @@ func (h Hash256) Value() (driver.Value, error) {
 
 // Scan scan value into HostSettings, implements sql.Scanner interface.
 func (hs *HostSettings) Scan(value interface{}) error {
-	bytes, ok := value.([]byte)
-	if !ok {
+	var bytes []byte
```

Comment:
was this necessary?

Reply:
SQLite started returning a `string` after the migration.